src/v/cluster/controller_backend.cc

```diff
@@ -42,18 +43,105 @@
 /// on every core, sharded
 namespace cluster {
 
+bool is_configuration_up_to_date(
+  model::node_id self,
+  const ss::lw_shared_ptr<partition>& partition,
+  const std::vector<model::broker_shard>& bs,
+  model::revision_id change_revision) {
+    auto group_cfg = partition->group_configuration();
+    auto configuration_committed = partition->get_latest_configuration_offset()
+                                   <= partition->committed_offset();
+
+    vlog(
```

Comment:
I think this log is not needed. this is just a static func that should just have a unit test.